code/src/components/GenreList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const GenreList = ({ genreList, setSelectedGenre, selectedGenre }) => {
+  const onOptionChangeHandler = (event) => {
+    setSelectedGenre(event.target.value)
+  }
+  return (
+    <select className="select-genre" value={selectedGenre} onChange={onOptionChangeHandler}>
```

Comment:
Well done, but you when I try to use the drop down on desktop, I can't see the different options since the text is light gray and the background is white.